README.md

```diff
@@ -94,13 +108,16 @@ Initializing the SDK should be done in native code inside your `AppDelegate` (iO
 ###### **Android:**
 ```java
 import com.adobe.marketing.mobile.AdobeCallback;
-import com.adobe.marketing.mobile.Identity;
 import com.adobe.marketing.mobile.InvalidInitException;
-import com.adobe.marketing.mobile.Lifecycle;
 import com.adobe.marketing.mobile.LoggingMode;
-import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Lifecycle;
```

Comment:
Add //MainApplication.java at the top of the import, so user know it is for MainApplication file?